lambdas/data-migration1/src/async-operations.ts

```diff
@@ -22,18 +21,32 @@ export const migrateAsyncOperationRecord = async (
   dynamoRecord: AWS.DynamoDB.DocumentClient.AttributeMap,
   knex: Knex
 ): Promise<void> => {
+  const asyncOperationPgModel = new AsyncOperationPgModel();
+
   // Use API model schema to validate record before processing
   Manager.recordIsValid(dynamoRecord, schemas.asyncOperation);
 
-  if (await doesRecordExist({ id: dynamoRecord.id }, knex, tableNames.asyncOperations)) {
+  let existingRecord;
+
+  try {
+    existingRecord = await asyncOperationPgModel.get(knex, { id: dynamoRecord.id });
+  } catch (error) {
+    // Swallow any RecordDoesNotExist errors and proceed with migration,
+    // otherwise re-throw the error
+    if (!(error instanceof RecordDoesNotExist)) {
+      throw error;
+    }
+  }
+
+  if (existingRecord && existingRecord.updated_at >= new Date(dynamoRecord.updatedAt)) {
```

Comment:
(I'll check this assertion later on, just commenting so it doesn't get dropped) -- we should carefully make sure the updatedAt values are identical on write.  I'm assuming that's part of this work.

Reply:
It's not, but it's part of this PR: https://github.com/nasa/cumulus/pull/2164

Reply:
Ah, nice 💯  - it's probably not that big a deal, but is this blocked merge on that then?

Reply:
It seems like merging wouldn't need to be blocked but Mark's PR would definitely need to be in before migrations are run. How does that feel?